scripts/global_constants.lua

```diff
@@ -22,6 +22,11 @@
 
 TUTORIAL_DUMMY_AMOUNT = 6
 
+-- Guardduty quest
+GUARDDUTY_BREADAMOUNT = 6
+GUARDDUTY_BREADAMOUNT_M = 4
+GUARDDUTY_BREADAMOUNT_F = 2
```

Comment:
Hm. I found this confusing first... I wondered what `M` meant without reading the quest code...
Maybe `GUARDDUTY_BREAD_AMOUNT_FOR_MALE_PRISONERS` would be more self-explaining... But I guess that might be my taste only...